wye/workshops/views.py

```diff
@@ -22,7 +22,7 @@ def get_context_data(self, *args, **kwargs):
         context['workshop_list'] = workshop_list
         context['workshop_feedback_pending'] = []
         context['workshop_in_queue'] = []
-        context['workshop_completed'] = []
+        context['workshop_completed'] = Workshop.objects.filter(status=WorkshopStatus.COMPLETED)
```

Comment:
Can you please add below query Workshop.objects.filter(status=WorkshopStatus.ACCEPTED)